cli_app/pipeline.py

```diff
@@ -46,6 +47,27 @@ def __init__(self, path: str, file_type: str, filename_format: str):
         self.total_file_count = 0
         self.source_file_count = 0
         self.target_file_count = 0
+        self.source_dated_dir = self.target_dated_dir = None
+
+    def validate_source_directory(self):
```

Comment:
here for creating source and target directory, two separate functions are not required , make one function and add into utility

Reply:
yes only single function needed